c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -3322,6 +3322,341 @@ JSObject *create_rs_proxy(JSContext *cx, HandleObject input_readable) {
 }
 } // namespace TransformStream
 
+/**
+ * Implementation of the WICG DecompressionStream builtin.
+ *
+ * All algorithm names and steps refer to spec algorithms defined at
+ * https://wicg.github.io/compression/#decompression-stream
+ */
+namespace DecompressionStream {
+namespace Slots {
+enum { Transform, Format, State, Buffer, Count };
+};
+
+enum class Format {
+  GZIP,
+  Deflate,
+  DeflateRaw,
+};
+
+// Using the same fixed encoding buffer size as Chromium, see
+// https://chromium.googlesource.com/chromium/src/+/457f48d3d8635c8bca077232471228d75290cc29/third_party/blink/renderer/modules/compression/deflate_transformer.cc#29
+const size_t BUFFER_SIZE = 16384;
```

Comment:
Might as well make this a `constexpr` :)

```suggestion
constexpr size_t BUFFER_SIZE = 16384;
```

Reply:
Nice, I've done that in a different pull-request: https://github.com/fastly/js-compute-runtime/pull/176/files#diff-c73cfed8b701bf02df88d204faa554410b6c531dac7a5b9c2109dc66506d3398R33